specification/frontdoor/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-frontdoor
+  package-version: 5.2.0
```

Comment:
Please remove this package-version property. We're now handling this in https://www.github.com/azure/azure-sdk-for-node.

Reply:
Nevermind. I'll take care of this file after this PR is merged.